File: backend_with_node/src/router/user.router.js
import { Router } from "express";
import { signIn, signUp ,signOut,checkCode} from "../controller/user.controller.js";
import Authjs from "../middleware/authJWT.js";
const userRouter= Router();

userRouter.route("/sign-up").post(signUp);
userRouter.route("/sign-in").post(signIn);
userRouter.route("/sign-out").post(Authjs,signOut);
userRouter.route("/checkcookie").get(checkCode);

export {userRouter} 

File: fast-api/app/routes/user_router.py
from fastapi import APIRouter, Depends
from app.controllers.user_controller import sign_up, sign_in, sign_out, check_cookie
from app.middleware.auth_jwt import verify_jwt

router = APIRouter(prefix="/api/user", tags=["User"])

router.post("/sign-up")(sign_up)
router.post("/sign-in")(sign_in)
router.post("/sign-out", dependencies=[Depends(verify_jwt)])(sign_out)
router.get("/checkcookie")(check_cookie)
